website/docs/language/block/output.mdx

```diff
@@ -0,0 +1,289 @@
+}
+```
+We recommend writing the description from the point of view of a module consumer to help consumers understand why the output exists and how to use it.
+
+#### Summary
+
+- Data type: String
+- Default: None
+
+### `sensitive`
+
+Add the `sensitive` argument to an `output` block to hide that block's value in the CLI output of a Terraform operation.
+
+```hcl
+output "admin_password" {
+  value     = <expression-to-evaluate>
+  sensitive = <boolean>
+}
+```
+
+If you set the `sensitive` argument to `true`, Terraform redacts that output value in plan and apply operations:
+
+```shell-session
+Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
+
+Outputs:
+
+admin_password = (sensitive value)
+```
+
+Terraform still records sensitive values in state, so anyone who can access your state data can access your sensitive values. For more information about safely storing sensitive data, refer to [Sensitive data in state](/terraform/language/state/sensitive-data).
+
+#### Summary
+
+- Data type: Boolean
+- Default: `false`
+
+### `ephemeral`
+
+-> **Note**: Ephemeral outputs are available in Terraform v1.10 and later.
+
+Add the `ephemeral` argument to `output` blocks in child modules to pass data between modules without persisting that data to state or plan files. You cannot add the `ephemeral` argument to `output` blocks in the root module.
+
+```hcl
+output "unique_password" {
+  value     = <expression-to-evaluate>
+  ephemeral = <boolean>
+}
+```
+
+Passing output values between child modules and the root module is useful for managing credentials, tokens, and other temporary resources that you do not want to store in state. If you enable the `ephemeral` argument on an `output` block, then your configuration must meet the following conditions:
+
+- The value of the `output` block must come from an ephemeral context.
+- You can only reference that output in other ephemeral contexts.
+
+The following ephemeral contexts can set values for output blocks with the `ephemeral` argument, and they can also reference output blocks with the `ephemeral` argument:
+- Another child module's ephemeral `output` block
+- A [write-only argument](/terraform/language/resources/ephemeral/write-only)
+- [Variables with the `ephemeral` argument](/terraform/language/values/variables#exclude-values-from-state)
+- [An `ephemeral` resource block](/terraform/language/resources/ephemeral)
+- Configuring providers in the [`provider` block](/terraform/language/providers)
+- In [provisioner](/terraform/language/resources/provisioners/syntax) and [connection](/terraform/language/resources/provisioners/connection) blocks
+
+#### Summary
+
+- Data type: Boolean
+- Default: `false`
+
+### `depends_on`
+
+The `depends_on` argument specifies an explicit dependency on another upstream resource for an `output` block. Terraform must complete all operations on an upstream resource before an `output` block can reference it.
```

Comment:
```suggestion
The `depends_on` argument specifies an explicit dependency on another upstream resource for an `output` block. Terraform will complete all operations on the upstream resource before it computes the value of an `output` block that depends on it.
```

We could also say "all upstream resources ... the values of output blocks that depend on it."